lib/grape/error_formatter/base.rb

```diff
@@ -35,10 +35,10 @@ def present(message, env)
 
         presenter = env[Grape::Env::API_ENDPOINT].entity_class_for_obj(message, present_options)
 
-        unless presenter || env[Grape::Env::RACK_ROUTING_ARGS].nil?
+        unless presenter || env[Grape::Env::API_ROUTING_ARGS].nil?
```

Comment:
Are we sure we need to change RACK_ROUTING_ARGS into API_ROUTING_ARGS? If we're going to change this maybe they should just be ROUTING_ARGS or GRAPE_ROUTING_ARGS?